src/client/pages/Thema/Profile/Profile-thema-config.ts

```diff
@@ -36,7 +36,7 @@ export const routeConfig = {
 
 export const profileLinks = {
   CHANGE_PERSONAL_DATA:
-    'https://www.amsterdam.nl/veelgevraagd/persoonlijke-gegevens-inzien-of-een-correctie-doorgeven-2bf85',
+    'https://www.amsterdam.nl/burgerzaken/fouten-gegevens-laten-aanpassen/',
   CHANGE_RESIDENT_COUNT:
     'https://www.amsterdam.nl/veelgevraagd/onjuiste-inschrijving-melden-ef918',
```

Comment:
Misschien op de toekomst vooruitlopen maar deze link gaat uiteindelijk hierheen en ik heb vaker die links met die 'willekeurige' character reeks aan het einde moeten vervangen zoals je nu ook doet.
```suggestion
    'https://www.amsterdam.nl/burgerzaken/verhuizen-inschrijving-briefadres/onjuiste-inschrijving-adres-melden',
```

Reply:
Goeie heb m aangepast